qiskit_experiments/library/tomography/qst_experiment.py

```diff
@@ -112,10 +130,10 @@ def _target_quantum_state(self) -> Union[Statevector, DensityMatrix]:
             # Circuit couldn't be simulated
             return None
 
-        if self._meas_qubits is None:
+        if self._meas_indices is None:
             return state
```

Comment:
I don't think `_meas_indices` can be None because of the line `self._meas_indices = tuple()` in the parent constructor. Should this instead just be checking false-like?

Same comment for state tomography.

Reply:
Yeah I think this should be `if not ...`